sktime/performance_metrics/forecasting/_functions.py

```diff
@@ -1495,37 +1495,39 @@ def mean_absolute_percentage_error(
     horizon_weight=None,
     multioutput="uniform_average",
     symmetric=False,
+    by_forecast=False,
     **kwargs,
 ):
-    """Mean absolute percentage error (MAPE) or symmetric version.
+    """Mean absolute percentage error (MAPE), symmetric MAPE (sMAPE), or by-forecast MAPE.
 
-    If ``symmetric`` is False then calculates MAPE and if ``symmetric`` is True
-    then calculates symmetric mean absolute percentage error (sMAPE). Both
-    MAPE and sMAPE output is non-negative floating point. The best value is 0.0.
+    If ``symmetric`` is False and ``by_forecast`` is False, calculates MAPE (mean absolute percentage error).
```

Comment:
great - I would suggest to add bullet points, because now it will be a bit confusing.